### ISSUE TITLE
After a successful locking flow, return the user to the veBAL page

### ISSUE THREAD

**Issue by pkattera** [balancer-labs/frontend-v2#1570]
**Describe the bug**
Currently, after a user successfully locks LP tokens in to veBAL, they are returned to the Locking flow. Since most people will max the lock amount, they immediately see an input error since the max amount will be exceeded. 

**To Reproduce**
1. Lock up LP tokens. 
2. See that you are returned to the lock screen with an error.

See screen recording:

https://www.loom.com/share/44229948dd9043f18489cef3a47d5464

**Expected behavior**
User should be taken back to the veBAL page, where they'll be able to see their updated veBAL balance. It's unlikely that the user will need to do another lock transaction immediately, so we'll save them from having to close the Lock screen to return to the veBAL page.

**Comment by garethfuller** [comment]
@pkattera When a lock is confirmed, you should actually see the success state in the preview. It shouldn't automatically redirect you anywhere or close the modal. When the modal is in the success state, the button at the bottom is a 'Return to veBAL page' link. So technically it does what you're suggesting already. I wonder if you maybe experienced this bug? https://github.com/balancer-labs/frontend-v2/issues/1565

**Comment by pkattera** [comment]
@garethfuller You are correct! Yeah, the button should be used upon successful locking. If the user taps outside the dialog, they should remain on the same page. Closing this ticket.
